docs/data/toolpad/getting-started/installation.md

```diff
@@ -25,3 +25,39 @@
    ```
 
 1. Build and deploy applications on http://localhost:3000/.
+
+## With Kubernetes
+
+### Prerequisites
+
+- A Kubernetes cluster (a local one via `kind` or a remote one)
+- The `kind` command line tool installed on your machine. Follow the [official instructions](https://kind.sigs.k8s.io/docs/user/quick-start/) to install `kind`.
```

Comment:
This should be agnostic from `kind`. "a cluster" is a prerequisite. We can give suggestions on how to create/obtain a cluster, but we shouldn't be opinionated about this